libs/atomic-elements/src/lib/components/GridSection/GridSection.tsx

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import ContextMenu from './ContextMenu';
+import { KsIcon } from '@kleeen/react/components';
+import MuiCheckbox from '@material-ui/core/Checkbox';
+import Paper from '@material-ui/core/Paper';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import MuiTableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import MuiTableHead from '@material-ui/core/TableHead';
+import MuiTableRow from '@material-ui/core/TableRow';
+import { styled } from '@material-ui/core/styles';
+import { Order, stableSort } from './stableSort';
+import useSort from './useSort';
+
+import './GridSection.scss';
+
+const TableHead = styled(MuiTableHead)({
+  '& .MuiTableCell-stickyHeader': {
+    backgroundColor: 'var(--table-header)',
+    color: 'var(--on-primary-color)',
+    fontSize: 'var(--tx-M)',
+    fontWeight: 'bold',
+    textTransform: 'uppercase',
+    '& .MuiButtonBase-root': {
+      '&:hover': {
+        color: 'var(--secondary-color)',
+      },
+    },
+    '& .kui-icon': {
+      height: 'var(--wh-3XS)',
+      width: 'var(--wh-2XS)',
+    },
+    '& .header-item': {
+      '& .sort-icon': {
+        '& .kui-icon': {
+          color: 'var(--on-primary-color)',
+        },
+      },
+    },
+  },
+});
+
+const Checkbox = styled(MuiCheckbox)({
+  paddingTop: '0',
+  paddingBottom: '0',
+  '& svg': {
+    fontSize: 'var(--wh-5XS)',
+  },
+  '&.MuiCheckbox-colorSecondary.Mui-checked': {
+    color: 'var(--on-primary-color)',
+    '&:hover': {
+      backgroundColor: 'var(--secondary-color)',
+    }
+  },
+});
+
+const TableRow = styled(MuiTableRow)({
+  '&:nth-of-type(odd)': {
+    backgroundColor: 'var(--table-odd-row-background-color)',
+  },
+  '&.MuiTableRow-root': {
+    '&.Mui-selected': {
+      backgroundColor: 'var(--secondary-color)',
+      '& .MuiTableCell-root': {
+        color: 'var(--on-primary-color)',
+        '& .context-menu-button': {
+          color: 'inherit',
+          '&:hover': {
+            backgroundColor: 'inherit',
+          }
+        },
+        '&:hover': {
+          '& .context-menu-button': {
+            color: 'var(--on-primary-color)',
+            backgroundColor: 'inherit',
+          },
+        },
+      },
+    },
+  },
+  '&.MuiTableRow-hover': {
+    '&:hover': {
+      backgroundColor: 'var(--secondary-color-variant)',
+      '& .MuiTableCell-root': {
+        '& .context-menu-button': {
+          color: 'var(--on-primary-color)',
+          backgroundColor: 'inherit',
+        },
+      },
+      '& .MuiCheckbox-colorSecondary': {
+        color: 'var(--on-primary-color)',
+      },
+    },
+  },
+});
+
+const TableCell = styled(MuiTableCell)({
+  color: 'var(--alt-mid-color)',
+  fontSize: 'var(--tx-S)',
+  paddingTop: 'var(--pm-2XS)',
+  paddingBottom: 'var(--pm-2XS)',
+  borderRight: '1px solid var(--grey_light_2x)',
```

Comment:
Can we use a variable here too?
```suggestion
  borderRight: 'var(something) solid var(--grey_light_2x)',
```

Reply:
It's an ongoing process. There are a lot of variables left at this point but @jcvalerio told us you need this PR. Variables are going to be added next PR